hiddenregions.xml

```diff
@@ -0,0 +1,79 @@
+<!--
```

Comment:
place this as a lua comment instead of an xml one, for easier porting to sm5, and so that if it crashes in this file, the line numbers line up

Reply:
you did this just now